packages/app/src/systems/Core/components/Layout/Layout.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ThemeUtilsCSS } from '@fuel-ui/css';
```

Comment:
It looks like the changes on the Layout have broken the scroll on the pages;
<img width="402" alt="Screenshot 2023-02-09 at 14 13 57" src="https://user-images.githubusercontent.com/3941923/217888698-d6e9e9bb-2899-4f8b-9b9c-1519ec386bd9.png">
<img width="453" alt="Screenshot 2023-02-09 at 14 13 22" src="https://user-images.githubusercontent.com/3941923/217888685-1922a0b3-9092-4584-bb4f-46d54ef9d25a.png">